packages/medusa/src/api/routes/admin/orders/list-orders.ts

```diff
@@ -0,0 +1,115 @@
+import { defaultAdminOrdersRelations, defaultAdminOrdersFields } from "."
+import { validator } from "../../../../utils/validator"
+import { IsOptional, IsString } from "class-validator"
+import _, { identity } from "lodash"
+import { OrderService } from "../../../../services"
+import { Selector } from "../../../../types/orders"
+
+/**
+ * @oas [get] /orders
+ * operationId: "GetOrders"
+ * summary: "List Orders"
+ * description: "Retrieves a list of Orders"
+ * x-authenticated: true
+ * parameters:
+ *   - (path) q {string} Query used for searching orders.
+ *   - (path) id {string} Id of the order to search for.
+ *   - (path) status {string[]} Status to search for.
+ *   - (path) fulfillment_status {string[]} Fulfillment status to search for.
+ *   - (path) payment_status {string[]} Payment status to search for.
+ *   - (path) display_id {string} Display id to search for.
+ *   - (path) cart_id {string} to search for.
+ *   - (path) customer_id {string} to search for.
+ *   - (path) email {string} to search for.
+ *   - (path) region_id {string} to search for.
+ *   - (path) currency_code {string} to search for.
+ *   - (path) tax_rate {string} to search for.
+ *   - (path) cancelled_at {DateComparisonOperator} Date comparison for when resulting orders was cancelled, i.e. less than, greater than etc.
+ *   - (path) created_at {DateComparisonOperator} Date comparison for when resulting orders was created, i.e. less than, greater than etc.
+ *   - (path) updated_at {DateComparisonOperator} Date comparison for when resulting orders was updated, i.e. less than, greater than etc.
+ *   - (path) offset {string} How many orders to skip in the result.
+ *   - (path) limit {string} Limit the number of orders returned.
+ *   - (path) expand {string} (Comma separated) Which fields should be expanded in each order of the result.
+ *   - (path) fields {string} (Comma separated) Which fields should be included in each order of the result.
```

Comment:
According to the [documentation](https://swagger.io/docs/specification/describing-parameters/#query-parameters) there's a difference between specifying something as part of the path and as query parameters. The documentation suggests using `in: query \n name: q  ... ` for query parameters. I guess the above states that it is actually part of the path, and not the query parameters (i.e., after `?`)